seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java

```diff
@@ -124,7 +124,7 @@ private void handleSubmitJob(HttpPostCommand httpPostCommand, String uri)
 
         boolean startWithSavePoint =
                 Boolean.parseBoolean(requestParams.get(RestConstant.IS_START_WITH_SAVE_POINT));
-        SeaTunnelServer seaTunnelServer = getSeaTunnelServer();
+        SeaTunnelServer seaTunnelServer = getSeaTunnelServer(false);
         RestJobExecutionEnvironment restJobExecutionEnvironment =
                 new RestJobExecutionEnvironment(
                         seaTunnelServer,
```

Comment:
> https://github.com/apache/seatunnel/blob/387e23b6492dc6ddc346fcf4a73889fcf9c1bc24/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java#L127-L147
> 
> If that's the case, it will never enter
> 
> 140 rows
> 
> If (seaTunnelServer==null){
> 
> This branch caused an error when submitting tasks

try again.

Reply:
I noticed that you rolled back to the first version of the code. I just tried using this version of the code, but I modified an additional line of code, which is 135 lines

hold

If (seaTunnelServer==null){

Changed to if (! SeaTunnelServer. isMasterNode()){

Currently, using HTTP to repeatedly submit the same task in cluster mode has not reported any errors, and the classLoader has not been added

Reply:
> I noticed that you rolled back to the first version of the code. I just tried using this version of the code, but I modified an additional line of code, which is 135 lines
> 
> hold `If (seaTunnelServer==null){`
> 
> Changed to `if (! SeaTunnelServer. isMasterNode()){`
> 
> Currently, using HTTP to repeatedly submit the same task in cluster mode has not reported any errors, and the classLoader has not been added

Yes you are right, I was superfluous.

Reply:
> I noticed that you rolled back to the first version of the code. I just tried using this version of the code, but I modified an additional line of code, which is 135 lines
> 
> hold `If (seaTunnelServer==null){`
> 
> Changed to `if (! SeaTunnelServer. isMasterNode()){`
> 
> Currently, using HTTP to repeatedly submit the same task in cluster mode has not reported any errors, and the classLoader has not been added

CC: @Hisoka-X